components/SearchSandbox/utils/datasearch-types.js

```diff
@@ -22,11 +16,17 @@ export default {
 		description: '',
 		input: 'string',
 	},
+	autosuggest: {
+		label: 'Auto Suggest',
+		description: 'This will enable search component to fetch suggestions as you type',
+		input: 'bool',
+		default: false,
+	},
 	highlight: {
 		label: 'Highlight results',
 		description: 'This will enable search highlighting',
 		input: 'bool',
-		default: false,
+		default: true,
```

Comment:
I'm not sure if this will get reflected in the Datasearch component by default. The way it works currently - these props don't get applied until unless "Save" is clicked on the datasearch modal. That's why all the defaults that you see here are actually Reactivesearch's defaults.